### PR TITLE
Helm chart v0.0.2 (electric boogaloo)

### PR DESCRIPTION
**Pull request by mgoodness**
Note that until we have a repo established, the installation instructions in the README are incorrect.

### CODE REVIEW

**Comment by joshrosso** [comment]
Awesome, thanks. Will review by Sunday.

**Comment by piclemx** [comment]
Nice work like always by @mgoodness  😃 !!

**Comment by mgoodness** [comment]
(Mostly) resolves #31. Pushing the chart to Quay & updating the README is #51.

File: helm/alb-ingress-controller/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+## Configuration
+
+The following tables lists the configurable parameters of the alb-ingress-controller chart and their default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`aws.accessKeyId` | If provided, AWS_ACCESS_KEY_ID environment variable will be set to this value | `""`
+`aws.accessSecretKey` | If provided, AWS_ACCESS_SECRET_KEY environment variable will be set to this value | `""`
+`aws.debug` | If true, enables logging on all outbound AWS API requests | `false`
+`aws.region` | (REQUIRED) AWS region in which this ingress controller will operate | `us-east-1`
```
Comment (joshrosso) [inline helm/alb-ingress-controller/README.md]: Any chance we can use `us-west-1` just to make it consistent with `alb-ingress-example` at https://github.com/coreos/alb-ingress-controller/blob/master/examples/alb-ingress-controller.yaml#L50-L51.

If not, I can switch the example to us-east-1

File: helm/alb-ingress-controller/values.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  accessSecretKey: ""
+
+  ## If true, enables logging on all outbound AWS API requests
+  #
+  debug: false
+
+  ## AWS region in which this ingress controller will operate
+  ## REQUIRED
+  #
+  region: us-east-1
```
Comment (joshrosso) [inline helm/alb-ingress-controller/values.yaml]: same question around us-west-1

**Pull request merged**